docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md

```diff
@@ -56,12 +60,13 @@ Grafana supports the following SAML 2.0 bindings:
 - From the Identity Provider (IdP) to the Service Provider (SP):
   - `HTTP-POST` binding
 
-In terms of security:
+### Security
 
-- Grafana supports signed and encrypted assertions.
-- Grafana does not support signed or encrypted requests.
+Grafana supports signed and encrypted assertions, and does _not_ support signed or encrypted requests.
```

Comment:
I think this is outdated, because Grafana supports signed requests.
```suggestion
Grafana supports signed and encrypted assertions, and does _not_ support encrypted requests.
```